source/guides/api-guidance/index.html.md.erb

```diff
@@ -43,48 +43,45 @@ API names are important, especially those APIs that are external as they are rep
 
 First, follow the GDS guidance here [When naming an API](https://www.gov.uk/guidance/gds-api-technical-and-data-standards#when-naming-and-hosting-your-api)
 
-### DfE API naming convention
 
-Use the below convention for external facing DfE APIs:
+## DfE API standards
 
-`<https://education-{api-name}.api.gov.uk/{resource>}`
+At DfE we inherit from the [GDS API technical and data standards](https://www.gov.uk/guidance/gds-api-technical-and-data-standards)
 
-Example:
+**You must follow these guidelines when developing an API for use in DfE. It helps us deliver secure, user centred API's that support our user centred digital services.**
 
-`<https://education-school-attendance.api.gov.uk/submission>`
+## How and where to publish a DfE API
 
-APIs must be named following these standards and guidelines, failure to do so may mean you will not be able to use an **api.gov.uk** domain making your API less discoverable for users.
+The best API designers also use their own APIs. Consider your new API in the context of existing DfE APIs and services to ensure you are not duplicating or confusing potential users. To help determine whether you should build a new API or add a resource to an existing one, check for existing APIs in your local repositories and the following public repositories:
 
-Once you have agreed a name you can get your domain from [Get an API domain on GOV.UK](https://www.gov.uk/guidance/get-an-api-domain-on-govuk).
+* [Find and Use an API](https://beta-find-and-use-an-api.education.gov.uk) A new service,currently in Public Beta, which provides a GOV.UK branded public facing front door for users to discover DfE APIs (see next section for further details)
+* [DfE-Digital GitHub organisation](https://github.com/DFE-Digital) Contains repositories with DfE's API source and documentation.
+* [GOV.UK API catalogue](https://www.api.gov.uk/#uk-government-apis) A new alpha service providing a cross-government view of available APIs. If you publish here you can list your API and link back to Find and Use an API.  The DfE has several disparate API catalogues and repositories. We are working to rationalize these using the Find and Use an API service to make it easier for potential users and publishers of APIs to connect.
 
-## DfE API standards
+## Find and Use an API
 
-At DfE we inherit from the [GDS API technical and data standards](https://www.gov.uk/guidance/gds-api-technical-and-data-standards)
-
-**You must follow these guidelines when developing an API for use in DfE. It helps us deliver secure, user centred API's that support our user centred digital services.**
+The DfE has a number of disparate API catalogues/repo's. We have recognised this and are working to rationalise these using the new [Find and Use an API](https://beta-find-and-use-an-api.education.gov.uk) service to make it easier and provide a consistent way for potential users and publishers of APIs to get connected.
```

Comment:
I know this predated your change but I don't think there should be an apostrophe in repos here.

Reply:
> I know this predated your change but I don't think there should be an apostrophe in repos here.

Agreed, I've updated the wording